docs/code_examples/systemd.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=Start Errbot chatbot
+After=network.service
+
+[Service]
+ExecStart=/path/to/my/virtualenv/bin/errbot --daemon --config /etc/errbot/config.py
```

Comment:
It's not necessary to let errbot daemonize itself. In fact, it is better not to (and won't work like this without also specifying `type=forking`).

It would also be good to incorporate some of the other items present in the supervisor config, namely setting the correct working dir, stop signal and locale.